### DISCUSSION TITLE
ERR_NETWORK_CHANGED

### DISCUSSION THREAD

**Discussion by ghost** [jef/streetmerchant#1181] [Help]
I have a VPN running to prevent me from getting banned.
What does this error mean? I think it might be from my IP address getting changed or something

**Comment by ghost** [comment]
Update: I am getting timeout error

"Navigation timeout of 3000ms exceeded"

**Comment by richstav** [comment]
Your public IP has changed and between the course of the switch on VPN and so network changed. Essentially, the same error you would get when you switch IP between the start and resolution of a query.
